### PR TITLE
LPS-141058 Call the remote services from the Account MVCActionCommand classes

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/DeleteAccountEntryMVCActionCommand.java [hunk 1]
```diff
@@ -49,7 +49,9 @@ protected void doProcessAction(
 		long[] accountEntryIds = ParamUtil.getLongValues(
 			actionRequest, "accountEntryIds");
 
-		_accountEntryLocalService.deleteAccountEntries(accountEntryIds);
+		for (long accountEntryId : accountEntryIds) {
+			_accountEntryService.deleteAccountEntry(accountEntryId);
+		}
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/DeleteAccountEntryMVCActionCommand.java:54]: Could we move this into a new service method?

**Review by drewbrokke — COMMENTED** [review]
Hey @pei-jung, this looks good, just a couple of questions. LMK what you think.

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/DeleteAccountEntryMVCActionCommand.java (hunk 1 above)
Comment (pei-jung) [reply modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/DeleteAccountEntryMVCActionCommand.java:54]: Yes, we can but that seems unusual. Bulk operations like add and delete are usually not supported in remote services.

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]
LGTM 👍

**Comment by epark18** [comment]
ci:stop

**Comment by epark18** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:stop

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by epark18** [comment]
Failures look unrelated to the pull.

**Comment by pei-jung** [comment]
@epark18 is this safe to forward manually? should I re-run the relevant tests?

**Comment by epark18** [comment]
> @epark18 is this safe to forward manually? should I re-run the relevant tests?

@pei-jung Since the failures looked unrelated, it should be okay. The semantic versioning failure was appearing for a lot of PR's at the same time this test was originally ran, but is currently passing now. Just to be safe, I can reevaluate as soon as the next [acceptance upstream build](https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/2540/) finishes.

**Comment by epark18** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**